### ISSUE TITLE
Publishing a standalone version on a gaming platform

### ISSUE THREAD

**Issue by Sworddragon** [jamesmgittins/incremancer#18]
In the past few months I was playing Incremancer twice but unfortunately todays web browsers involve several difficulties to make this a good experience. Personally for me strong anti-tracking settings (permanent PBM in Firefox, anti-fingerprinting, etc.) don't allow for an easy persistent save (export/import is possible but still tedious to do on a regular basis) and mixing different browsing contexts in such a setup is not desired as well. Also from my experience the game performance in web browsers is usually worse if a game is available via the web and Steam, but maybe this is just a Firefox vs. Google Chrome issue.

For others and also me the main issue is that energy saving mechanisms (background throttling, window occlusion, etc.) mess with the games speed and tamper the game logic and browsers have no good ways to mitigate this. On doing a more detailed test with Google Chrome by disabling those optimizations via command line arguments I was not able to achieve more than 70% of the original game speed when the game was minimized plus that using such options for a niche case basically naturally triggered a not so well-tested scenario by revealing a major bug in Google Chrome: Memory is not freed anymore for the tab and raises up pretty fast causing the tab to crash after around 2 hours due to being out of memory.

I made a report including my Google Chrome comparison on the Firefox tracker (since it is my main browser) to hopefully get better game support here in the future - but I'm not raising my hopes up too high. In case anybody is interested into reading this the link is https://bugzilla.mozilla.org/show_bug.cgi?id=1923253 .

The only serious answer I currently see is having such games as standalone version on a gaming platform like Steam to avoid all those issues but unfortunately from my experience this is barely done in the case of (idle-)webgames.

Also I'm actually surprised nobody made this suggestion already in this tracker here especially since Incremancer is the 2nd-highest rated idle-game on Kongregate. So I'm doing the step and just requesting to publish a Steam (or another gaming platform like GOG) release. And as usual for such requests: This could also be an opportunity to make a bit of money from it - even if the game is kept free and only a DLC supporter pack as donation option is being used.

**Comment by Realswitzer** [comment]
Agreed, easiest way seems to be using [nwutils/Web2Executable](https://github.com/nwutils/Web2Executable), however publishing to Steam has a $70 fee and given the lack of contributions to Incremancer in 4 years as of writing, I doubt this will ever happen.
